airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -84,6 +117,7 @@ class ClusterGenerator:
         to create the cluster. (templated)
     :param num_workers: The # of workers to spin up. If set to zero will
         spin up cluster in a single node mode
+    :param min_num_workers: The minimum number of primary worker instances to create.
```

Comment:
Its not immediately clear how this value interacts with `num_workers`, might be worth elaborating a bit more?

From the API docs:
> Example: Cluster creation request with num_instances = 5 and min_num_instances = 3:
>
> If 4 VMs are created and 1 instance fails, the failed VM is deleted. The cluster is resized to 4 instances and placed in a RUNNING state.
> If 2 instances are created and 3 instances fail, the cluster in placed in an ERROR state. The failed VMs are not deleted.

Could we add some of that context here? I don't have any strong opinions on the exact wording.

Reply:
Updated.